db/seeds.rb

```diff
@@ -1,5 +1 @@
-# Loads seed data out of default dir
-default_path = File.join(File.dirname(__FILE__), 'default')
-
-Rake::Task['db:load_dir'].reenable
-Rake::Task['db:load_dir'].invoke(default_path)
+require_relative 'default/users.rb'
```

Comment:
There are debugging/testing scenarios where this potentially could be useful, but yes, not on a regular basis, I agree. I actually don't care that much, just wanted to keep the former behavior.